src/charm.py

```diff
@@ -194,6 +184,32 @@ def mimir_worker_relations(self) -> List[ops.Relation]:
         """Returns the list of worker relations."""
         return self.model.relations.get("mimir_worker", [])
 
+    @property
+    def workers_scrape_jobs(self) -> List[Dict[str, str]]:
+        """Scrape jobs for the Mimir workers."""
+        scrape_jobs = []
+        worker_topologies = self.cluster_provider.gather_topology()
+        for worker in worker_topologies:
+            job = {
+                "static_configs": [
+                    {
+                        "targets": [f"{worker['address']}:8080"],
```

Comment:
Metrics endpoint for mimir is under `/metrics` - https://grafana.com/docs/mimir/latest/references/http-api/#metrics. Is scrape job config clever enough to figure it out by itself or do we need to add it?

Reply:
Seems like yes.